src/main/java/com/denizenscript/depenizen/bukkit/events/towny/PlayerJoinsTownScriptEvent.java

```diff
@@ -0,0 +1,58 @@
+package com.denizenscript.depenizen.bukkit.events.towny;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.depenizen.bukkit.objects.towny.TownTag;
+import com.palmergames.bukkit.towny.event.TownAddResidentEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class PlayerJoinsTownScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // towny player joins town
+    //
+    // @Cancellable false
+    //
+    // @Triggers when a player joins a Towny town.
+    //
+    // @Context
+    // <context.town> Returns town the player is joining in.
```

Comment:
English grammar: should be `returns the town` - lowercase `r` and include `the`
also remove `in` from the end.